src/components/3d_editor/advanced_geometry/python_transformation/MaterialsSelector.tsx

```diff
@@ -31,7 +32,7 @@ function MaterialsSelector(props: MaterialsSelectorProps) {
         <Autocomplete
             multiple
             id="materials-autocomplete"
-            data-tid="materials-selector"
+            data-tid={testId ?? ""}
```

Comment:
![dafuq](https://user-images.githubusercontent.com/721112/209405526-d56fc0a1-a610-4390-b75f-bb35534eed9a.gif)

Reply:
Since there are at least two materials selectors we need to pass separate testIds to them. But the line itself is weird, agree.